client/.eslintrc.js

```diff
@@ -23,6 +23,11 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    commonjs: true
```

Comment:
We might not want to allow using this env everywhere. Was there too many files where you would've had to add an explicit `/* eslint-env commonjs */` in the top?